### PR TITLE
minor design tweaking and some new modal boxes

### PR DESCRIPTION
**Pull request by FailXSiegen**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,13 +18,12 @@
     "jquery": "^3.5.1",
     "popper.js": "^1.16.1",
     "ramda": "^0.27.1",
+    "serve": "^11.3.2",
```
Comment (0m3gaC0d3) [inline package.json:21]: Why do you add this dependency?

Comment (FailXSiegen) [reply package.json:21]: i might have used npm install locally...

Comment (FailXSiegen) [reply package.json:21]: but i didn't add any package on purpose

Comment (FailXSiegen) [reply package.json:21]: now i understand this shit XD i didn't write "npm run serve" correctly and add the "serve" to the package... lolololol

**Review by 0m3gaC0d3 — APPROVED** [review]

**Pull request merged**